src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,30 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+
+        int userScore = -1;
+
+        if (records == null) {
+            throw new UserNotFoundException("User with given email doesn't exist");
+
```

Comment:
for this task this checking is redundant - let's remove it

Reply:
done